src/main/java/seedu/address/logic/commands/ConcreteCommand.java

```diff
@@ -0,0 +1,16 @@
+package seedu.address.logic.commands;
+
+import seedu.address.model.Model;
+
+/**
+ * Represents a command which can be undone by the undo command.
+ */
+public abstract class ConcreteCommand extends Command {
```

Comment:
nice job on creating a concreteCommand class for undo command that can unit the method they use for undo